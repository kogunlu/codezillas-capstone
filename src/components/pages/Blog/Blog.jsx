import React from 'react'
import meditation from "./meditation.png" 
import trauma from "./trauma.png"
import anxiety from "./anxiety.png"


function Blog() {
  return (
    
    <div className="flex flex-col justify-center items-center p-6 m-14">      
    <br/> 
    <img className= "justify-center rounded" src={meditation} alt="meditation"/> 
   <h1 className=''>MENTAL HEALTH AND TECHNOLOGY </h1>
   <br/> <br /> 
   <p className='text-base'>Technology and mental well-being might seem like profoundly antagonistic terms. However, there are excellent digital tools to help you reach mindfulness and to practice meditation. You can even use smartphone apps, virtual reality solutions and digital devices as a form of relieving stressing and achieving digital detox – as, after a while, you will be able to practice all types of anti-anxiety skills on your own. So, let The Medical Futurist show you how you can enhance your mental and emotional health with the help of technology. </p>
   <br/> 
   <h3 className='uppercase'>THE NEED FOR TAKING CARE OF OUR EMOTIONAL HEALTH </h3> 
   <p className='text-base space-x-0'>“Just as we observe physical hygiene to stay well, we need to cultivate a kind of emotional hygiene too,” wrote the Dalai Lama on his Twitter-feed. There are three crucial points around His Holiness’ statement. First and foremost, the popularity of health-enhancing, physically active lifestyle is soaring worldwide. Nevertheless, there’s always room for improvement: the WHO says that globally, 23 percent of adults and 81 percent of school-going adolescents are not active enough. According to the organization, adults aged 18-64 should do at least 150 minutes of moderately intense physical activity each week, which is not unaccomplishable. Moreover, the numerically and statistically driven 21st-century mind could take a great inspiration and power from the various fitness trackers and wearables.

</p> <p className='text-base space-x-0'>Secondly, emotional hygiene seems to be thrown into the shade nowadays. Somehow, the culture of busyness prevents us from squeezing some time for mental health into our packed schedules. Information is pouring on us from millions of communication channels; we are connecting with hundreds of ‘friends’ on social media, we are obsessed with data due to the fear of missing out and time management slices up our days into meetings and to-do-lists.

</p> <p className='text-base space-x-0'>The pressure to perform and the stress to achieve is the air every busy-bee is breathing. However, turbulent lifestyle comes with a price. No wonder so many people have mental health issues. In the US, that means one in five adults – more than 40 million Americans! On the old continent, 27 percent of the population has to cope with mental disorders, which means 83 million struggling Europeans. In Asia, the prevalence of depression is 20 percent in Thailand, 19.9 percent in Taiwan, 19.4 percent in Korea, 17.5 in Malaysia and 16.5 percent in China.</p> 
<br/> 
<h3 className='uppercase'>Sign up for The Healing blog</h3>
<h6 className='uppercase text-sm'>A weekly, ad-free Blog that helps you stay in the know.</h6>
<h3 className='uppercase'>Recommended for you</h3> 
<div className='flex flex-row'> <img className= "justify-center m-10 w-auto h-auto" title='HEAL FROM TRAUMA' src={trauma} alt="trauma"/><img className= "justify-center m-10 w-auto h-auto" src={anxiety} alt="anxiety"/> </div>
  </div>  
       
  ) 
} 
 
export default Blog   