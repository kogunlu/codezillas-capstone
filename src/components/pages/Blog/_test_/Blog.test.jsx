  import React from "react";
  import renderer from "react-test-renderer";
  import { BrowserRouter } from "react-router-dom"
  import Blog from "../Blog";
  
  it('Renders correctly for Blog Component', () => {
      const tree = renderer
      .create(
      <BrowserRouter>
      <Blog />
      </BrowserRouter> 
      )
      .toJSON();
      expect(tree).toMatchSnapshot();
    }); 