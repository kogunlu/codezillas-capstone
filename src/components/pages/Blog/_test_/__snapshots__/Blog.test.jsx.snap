// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly for Blog Component 1`] = `
<div
  className=" lg:mx-auto lg:text-justify "
>
  <img
    alt="meditation"
    className=" items-start lg:justify-center rounded mx-auto py-7  "
    src="meditation.png"
  />
  <div
    className="max-w-[800px]  w-full  mx-auto  flex flex-col justify-center px-3"
  >
    <h1
      className=" my-3 text-bold text-3xl"
    >
      MENTAL HEALTH AND TECHNOLOGY
    </h1>
    <p
      className="uppercase mb-4"
    >
      Technology and mental well-being might seem like profoundly antagonistic terms. However, there are excellent digital tools to help you reach mindfulness and to practice meditation. You can even use smartphone apps, virtual reality solutions and digital devices as a form of relieving stressing and achieving digital detox – as, after a while, you will be able to practice all types of anti-anxiety skills on your own. So, let The Medical Futurist show you how you can enhance your mental and emotional health with the help of technology.
    </p>
    <h3
      className="uppercase py-5 text-xl "
    >
      THE NEED FOR TAKING CARE OF OUR EMOTIONAL HEALTH
    </h3>
    <p
      className="uppercase"
    >
      “Just as we observe physical hygiene to stay well, we need to cultivate a kind of emotional hygiene too,” wrote the Dalai Lama on his Twitter-feed. There are three crucial points around His Holiness’ statement. First and foremost, the popularity of health-enhancing, physically active lifestyle is soaring worldwide. Nevertheless, there’s always room for improvement: the WHO says that globally, 23 percent of adults and 81 percent of school-going adolescents are not active enough. According to the organization, adults aged 18-64 should do at least 150 minutes of moderately intense physical activity each week, which is not unaccomplishable. Moreover, the numerically and statistically driven 21st-century mind could take a great inspiration and power from the various fitness trackers and wearables.
    </p>
     
    <p
      className="uppercase py-6"
    >
      Secondly, emotional hygiene seems to be thrown into the shade nowadays. Somehow, the culture of busyness prevents us from squeezing some time for mental health into our packed schedules. Information is pouring on us from millions of communication channels; we are connecting with hundreds of ‘friends’ on social media, we are obsessed with data due to the fear of missing out and time management slices up our days into meetings and to-do-lists.
    </p>
    <p
      className="uppercase"
    >
      The pressure to perform and the stress to achieve is the air every busy-bee is breathing. However, turbulent lifestyle comes with a price. No wonder so many people have mental health issues. In the US, that means one in five adults – more than 40 million Americans! On the old continent, 27 percent of the population has to cope with mental disorders, which means 83 million struggling Europeans. In Asia, the prevalence of depression is 20 percent in Thailand, 19.9 percent in Taiwan, 19.4 percent in Korea, 17.5 in Malaysia and 16.5 percent in China.
    </p>
  </div>
  <div
    className=" w-full max-w-[800px] mx-auto mt-12 mb-24 flex flex-col justify-center px-3"
  >
    <h3
      className="uppercase lg:text-xl font-weight:600 sm:m-0"
    >
      Sign up for The Healing blog
    </h3>
    <h6
      className="uppercase mb-4 py-2 font-weight:600"
    >
      A weekly, ad-free Blog that helps you stay in the know.
    </h6>
    <div
      className="flex justify-start items-center"
    >
      <input
        className="border border-gray-400 rounded-l-md h-10 pl-2 focus:outline-none "
        onChange={[Function]}
        placeholder="Enter your e-mail"
        type="email"
        value=""
      />
      <button
        className="h-10  bg-cyan-400 pl-1 pr-2 rounded-r-md border-y border-r border-gray-400 hover:bg-cyan-300 text-xl"
        onClick={[Function]}
        type="button"
      >
        <svg
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          strokeWidth="0"
          style={
            Object {
              "color": undefined,
            }
          }
          viewBox="0 0 448 512"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M446.7 98.6l-67.6 318.8c-5.1 22.5-18.4 28.1-37.3 17.5l-103-75.9-49.7 47.8c-5.5 5.5-10.1 10.1-20.7 10.1l7.4-104.9 190.9-172.5c8.3-7.4-1.8-11.5-12.9-4.1L117.8 284 16.2 252.2c-22.1-6.9-22.5-22.1 4.6-32.7L418.2 66.4c18.4-6.9 34.5 4.1 28.5 32.2z"
          />
        </svg>
      </button>
    </div>
  </div>
  <div
    className="flex justify-center mb-10 "
  >
    <div
      className="w-5/6 flex flex-col lg:flex-row justify-between gap-4"
    >
      <div
        className="w-full lg:w-6/12 relative"
      >
        <img
          alt="trauma"
          className=" "
          src="trauma.png"
          title="HEAL FROM TRAUMA"
        />
        <span
          className="absolute bottom-1/4 left-5 w-max "
        >
          <p
            className="text-white text-start text-sm md:text-lg font-bold bg-slate-400 opacity-90  "
          >
            HEAL FROM
          </p>
          <p
            className="text-white text-start text-sm md:text-lg font-bold bg-slate-400 opacity-90 mt-1 "
          >
            TRAUMA
          </p>
        </span>
      </div>
      <div
        className="w-full lg:w-6/12 relative"
      >
        <img
          alt="anxiety"
          className=""
          src="anxiety.png"
        />
        <span
          className="absolute top-5 left-3 w-2/6"
        >
          <p
            className="font-medium text-sm md:text-lg text-start"
          >
            TELL ME ALL I NEED TO KNOW ABOUT ANXIETY
          </p>
        </span>
      </div>
    </div>
  </div>
</div>
`;
